static/src/javascripts/projects/common/modules/commercial/membership-engagement-banner.js

```diff
@@ -94,7 +94,11 @@ const deriveBannerParams = (location: string): ?EngagementBannerParams => {
     const campaignId = userTest ? userTest.campaignId : undefined;
     const userVariant = getUserVariant(userTest);
 
-    if (userVariant && userVariant.blockEngagementBanner) {
+    if (
+        userVariant &&
+        userVariant.options &&
+        userVariant.options.blockEngagementBanner
```

Comment:
The one piece of implementation I have changed. It looks like [`blockEngagementBanner`](https://github.com/guardian/frontend/blob/8b36918c95d4c983232774cf8845d361a83a1b61/static/src/javascripts/projects/common/modules/commercial/contributions-utilities.js#L243) exists on `Variant.options`, not directly on `Variant`. Would this be fair to say, @Mullefa?

Reply:
yes, that's correct. Thanks!